taccsite_cms/settings.py

```diff
@@ -539,15 +547,29 @@ def get_subdirs_as_module_names(path):
 
 # https://github.com/django-cms/djangocms-style
 DJANGOCMS_STYLE_CHOICES = [
-    # https://cep.tacc.utexas.edu/design-system/ui-patterns/o-section/
+    # https://tacc.utexas.edu/static/ui/components/detail/c-card--default.html
+    'card',
+    'card--plain',
+    'card--standard',
+    'card--image-top',
+    'card--image-bottom',
+    'card--image-right',
+    'card--image-left',
+    # https://tacc.utexas.edu/static/ui/components/detail/o-section.html
+    'section',
+    'section--light',
+    'section--muted',
+    'section--dark',
+    'o-section',
     'o-section o-section--style-light',
+    'o-section o-section--style-muted',
     'o-section o-section--style-dark',
     # https://cep.tacc.utexas.edu/design-system/ui-patterns/c-callout/
     'c-callout',
     # https://cep.tacc.utexas.edu/design-system/ui-patterns/c-recognition/
     'c-recognition c-recognition--style-light',
     'c-recognition c-recognition--style-dark',
-    # https://cep.tacc.utexas.edu/design-system/ui-patterns/c-nav/
+    # https://ecepalliance.org/alliance-members/
```

Comment:
Do we need this line?

Reply:
**No, we don't.** How about I replace it with a URL that is more clearly a link to a UI pattern?[^1] Or, I drop them all. Or maybe just have one "good" link… I'm liking this idea.

[^1]: This, and other URL comments in this array, point to the only (yet sometimes unobvious) official(-ish) instances of the UI pattern the class supports. This one is especially unclear, because it does not mention UI.

Reply:
I removed all such URLs in this array.

<sup>Those links are relevant to TACC/Core-Styles now, which maintains their code (but sometimes lacks example usage, which is what those URLs were).</sup>